core/classes/Locale/LanguageKey.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Represents a valid language key that exists in Nameless.
+ *
+ * @package NamelessMC\Locale
+ * @author Aberdeener
+ * @version 2.3.0
+ * @license MIT
+ */
+class LanguageKey
+{
+    private string $section;
+    private string $term;
+    private string $module;
+    private array $variables;
+
+    public function __construct(string $section, string $term, array $variables = [], ?string $module = 'core')
+    {
+        $language = new Language($module, 'en_UK');
```

Comment:
So this class is only intended for Notification class as you force english here? As i see the Notification class doing the part of getting the term for user language

Reply:
Yes this is to ensure that the term passed to `LanguageKey` exists in at least the base language as a sort of typo check